controllers/user.js

```diff
@@ -61,38 +79,47 @@ async function add(req, res) {
   });
 }
 
-const bundleVerifiedEmails = async(email, emailSha1, ifPrimary, allBreaches) => {
+const bundleVerifiedEmails = async(email, emailSha1, ifPrimary, id, verificationStatus, allBreaches) => {
```

Comment:
Let's consolidate on either function declarations unless we need the anonymous or immediate properties of expressions.
```
async function bundleVerifiedEmails(email, emailSha1, ifPrimary, id, verificationStatus, allBreaches) {
...
}
```